security/.project

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there a reason to check-in IDE specific files, such as this Eclipse project file?

Reply:
Just the project file which both IntelliJ and Eclipse support 😃